stencil.config.ts

```diff
@@ -1,11 +1,20 @@
 import { Config } from '@stencil/core';
 import { sass } from '@stencil/sass';
 import { kompendium } from 'kompendium';
+import nodePolyfills from 'rollup-plugin-node-polyfills';
+
 import guides from './guides';
 
 export const config: Config = {
     hashFileNames: false,
     namespace: 'lime-elements',
+    plugins: [sass()],
+    rollupPlugins: {
+        after: [nodePolyfills()],
+    },
+    nodeResolve: {
+        preferBuiltins: true,
+    },
```

Comment:
Did we still need this? If so, should it not be needed in the other config file as well? :slightly_smiling_face:

Reply:
We don't need it anymore so I removed it.